cfgov/cfgov/urls.py

```diff
@@ -418,6 +418,13 @@ def flagged_wagtail_template_view(flag_name, template_name):
     flagged_url('REGULATIONS3K',
                 r'^regulations3k/',
                 include_if_app_enabled('regulations3k', 'regulations3k.urls')),
+    flagged_url('REGULATIONS3K',
+                r'^eregs-service-worker.js',
+                TemplateView.as_view(
+                template_name='regulations3k/eregs-service-worker.js',
+                content_type='application/javascript'),
+                name='eregs-service-worker.js'),
+
```

Comment:
@willbarton @higs4281 Is there a better way to serve the service worker? It needs to be accessible at `https://consumerfinance.gov/eregs-service-worker.js`.